cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
+
 	clusterUp(&config, ip)
 	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
 }
 
+func automountHostfolders(driver drivers.Driver) {
+	if hostfolder.IsAutoMount() && hostfolder.IsHostfoldersDefined(false) {
+		hostfolder.MountHostfolders(driver)
```

Comment:
> Testing with driver is quite easy... already pushed a change for this.

Sure, but there are much more mocks involved . Also I would like at some stage look into providing an alternative to the current ssh commander which allows streaming of responses (if possible, not sure). I think this would be easier if we decouple from the driver.